consensus/src/pending_votes.rs

```diff
@@ -385,26 +408,33 @@ fn hash_index_to_str(hash_index: usize) -> String {
 
 impl fmt::Display for PendingVotes {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        // collect votes per ledger info
-        let votes = self
-            .li_digest_to_votes
-            .iter()
-            .map(|(li_digest, (_, li))| (li_digest, li.signatures().keys().collect::<Vec<_>>()))
-            .collect::<BTreeMap<_, _>>();
+        write!(f, "PendingVotes: [")?;
+
+        for (li_digest, (_, status)) in self.li_digest_to_votes.iter() {
+            match status {
+                VoteStatus::EnoughVotes(_li) => {
+                    write!(f, "LI {} has aggregated QC", li_digest)?;
+                },
+                VoteStatus::NotEnoughVotes(li) => {
+                    write!(
+                        f,
+                        "LI {} has {} verified votes {:?}, {} unverified votes {:?}",
```

Comment:
please don't use debug format, it spams the log

Reply:
Done.